go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -144,6 +144,7 @@ func init() {
 	flag.BoolVar(&enableHeartbeat, "heartbeat_enable", false, "If true, vttablet records (if master) or checks (if replica) the current time of a replication heartbeat in the table _vt.heartbeat. The result is used to inform the serving state of the vttablet via healthchecks.")
 	flag.DurationVar(&heartbeatInterval, "heartbeat_interval", 1*time.Second, "How frequently to read and write replication heartbeat.")
 	flag.BoolVar(&currentConfig.EnableLagThrottler, "enable-lag-throttler", defaultConfig.EnableLagThrottler, "If true, vttablet will run a throttler service, and will implicitly enable heartbeats")
+	flag.BoolVar(&currentConfig.EnableLagThrottler, "enable_lag_throttler", currentConfig.EnableLagThrottler, "synonym to -enable-lag-throttler")
```

Comment:
If both are specified, which one wins?
In order to standardize on underscore, should we document the `_` version of the flag and call the other one a synonym?

Reply:
in the above case, underscores override dashes. Or, actually, I think that just depends on the order specified on the command line (with any single-valued flag, you can specify it as many times you want on the CLI and the last one wins it all).

> In order to standardize on underscore, should we document the _ version of the flag and call the other one a synonym?

Agreed.

Reply:
BTW confirmed, last flag specified on the command line wins, regardless of which order the flags are created in code.